youtube_dl/extractor/common.py

```diff
@@ -1812,10 +1812,10 @@ def _m3u8_meta_format(self, m3u8_url, ext=None, preference=None, m3u8_id=None):
         }
 
     def _report_ignoring_subs(self, name):
-        self.report_warning(bug_reports_message(
+        self.report_warning(
             'Ignoring subtitle tracks found in the {0} manifest; '
-            'if any subtitle tracks are missing,'.format(name)
-        ), only_once=True)
+            'if any subtitle tracks are missing, {1}'.format(name, bug_reports_message()),
+            only_once=True)
```

Comment:
Revert and fix `utils.bug_reports_message()` (1):
```suggestion
        self.report_warning(bug_reports_message(
            'Ignoring subtitle tracks found in the {0} manifest; '
            'if any subtitle tracks are missing,'.format(name)
        ), only_once=True)
```